Translation corrections (#435)
07th-mod/higurashi-console-arcs: Update/ura_009.txt

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#7d746d>山沖</color>", NULL, "<color=#7d746d>Yamaoki</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 30, "ps3/s14/30/vt4bb_yamaoki1115", 256, TRUE);
	OutputLine(NULL, "「…言わずもがなと思って黙ってきたが、あえて伝えよう。",

> 			NULL, "\"...I thought this would be better left unsaid, but I'll tell you anyway.", Line_WaitForInput);

Reviewer: ```suggestion
			NULL, "\"...I feel this is better left unsaid, but I'll tell you anyway.", Line_WaitForInput);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#7d746d>山沖</color>", NULL, "<color=#7d746d>Yamaoki</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 30, "ps3/s14/30/vt4bb_yamaoki1122", 256, TRUE);
	OutputLine(NULL, "「『新しい捜査形態』を体系づけた立役者の一人として、君は東京に凱旋する…という筋書きでな。",
			NULL, "\"As one of the key players in this \"new form of investigation,\" you'll have to return to Tokyo... That's how it'll play out.", Line_WaitForInput);
	ModPlayVoiceLS(4, 30, "ps3/s14/30/vt4bb_yamaoki1123", 256, TRUE);
	OutputLine(NULL, "科警研の連中の動きを、逆に利用したわけだ。",
			NULL, " NRIPS activities will have been used against them. ", GetGlobalFlag(GLinemodeSp));
	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#7d746d>山沖</color>", NULL, "<color=#7d746d>Yamaoki</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 30, "ps3/s14/30/vt4bb_yamaoki1124", 256, TRUE);
	OutputLine(NULL, "ＴＶ局側も大筋で、その代替案に賛成らしい」",

> 			NULL, "And the TV stations seem to be in favor with that alternative approach.\"", Line_Normal);

Reviewer: ```suggestion
			NULL, "And the TV stations seem to be in favor of that alternative approach.\"", Line_Normal);
```

//rしかし、前野室長が彼女を守るためにそこまで心を砕いてくれていたとは、あまりにも意外なことだった。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "しかし、前野室長が彼女を守るためにそこまで心を砕いてくれていたとは、あまりにも意外なことだった。",

> 			NULL, "But she was surprised to learn that Director Maeno was that willing to wrack his brains so much to protect her.", Line_Normal);

Reviewer: ```suggestion
			NULL, "But she was surprised to learn that Director Maeno was that willing to wrack his brain so far to protect her.", Line_Normal);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#7d746d>山沖</color>", NULL, "<color=#7d746d>Yamaoki</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 30, "ps3/s14/30/vt4bb_yamaoki1125", 256, TRUE);
	OutputLine(NULL, "「やつも長い間、理解者に恵まれず苦労したクチだからな。",

> 			NULL, "\"Because he too has worked for a long time without having anyone to understand him.", Line_WaitForInput);

Reviewer: ```suggestion
			NULL, "\"Because he too had worked for a long time without having anyone to understand him.", Line_WaitForInput);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#7d746d>山沖</color>", NULL, "<color=#7d746d>Yamaoki</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 30, "ps3/s14/30/vt4bb_yamaoki1132", 256, TRUE);
	OutputLine(NULL, "「…だが、机上の空論で固まった人間を作り出さないためには、あえて皆が目を背ける現実で苦労させることも必要だ。",

> 			NULL, "\"...But, in order avoid creating those stuck in unpracticed theories, it's necessary to make them face a difficult reality that everyone else would turn away from.", Line_WaitForInput);

Reviewer: ```suggestion
			NULL, "\"...But, in order avoid creating people fixated on unpracticed theories, it's necessary to make them face a difficult reality that everyone else would turn away from.", Line_WaitForInput);
```

//r自分は、本当に恵まれている。…それは、彼女の本心だった。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "自分は、本当に恵まれている。…それは、彼女の本心だった。",

> 			NULL, "She truly was blessed. ...That was how she felt deep down.", Line_Normal);

Reviewer: ```suggestion
			NULL, "She truly was blessed. ...Deep in her heart, she felt that was the truth.", Line_Normal);
```

			NULL, "\"In order to act freely of all ties of obligation... That was how my father saw justice.", Line_WaitForInput);
	ModPlayVoiceLS(4, 28, "ps3/s14/28/vt4bb_tomoe1898", 256, TRUE);
	OutputLine(NULL, "だけど、父のそんな思いは組織にとって、ただの独りよがりな、愚かなものだったのでしょうか…？」",

> 			NULL, " But were his feelings foolish and selfish to the police...?\"", Line_Normal);

Reviewer: ```suggestion
			NULL, " But were his feelings foolish and selfish in the eyes of to the police...?\"", Line_Normal);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#7d746d>山沖</color>", NULL, "<color=#7d746d>Yamaoki</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 30, "ps3/s14/30/vt4bb_yamaoki1160", 256, TRUE);
	OutputLine(NULL, "「幸い死者こそ出なかった…というのは建前で、実際には患者が２人、その傷がもとで死亡している」",

> 			NULL, "\"Fortunately, no one died in the incident... At least, that's the official story, but the truth is, two patients did die from their wounds.\"", Line_Normal);

Reviewer: ```suggestion
			NULL, "\"Fortunately, no one died in the incident... At least, that's the official story, but the truth is two patients did die from their wounds.\"", Line_Normal);
```

//rその見返りと報いも一身に受け止めて、傷つくのも倒れるのも自身のみだった。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "その見返りと報いも一身に受け止めて、傷つくのも倒れるのも自身のみだった。",

> 			NULL, "Accepting the collateral and retribution alone, making sure only she got hurt and dragged down.", Line_Normal);

Reviewer: ```suggestion
			NULL, "Accepting the collateral and retribution alone, making sure nobody else got hurt and dragged down.", Line_Normal);
```
